html/cross-origin-opener-policy/javascript-url.https.html

```diff
@@ -0,0 +1,257 @@
+<!DOCTYPE html>
+<title>Cross-Origin-Opener-Policy and a "javascript:" URL popup</title>
+<meta charset="utf-8">
+<script src="/resources/testharness.js"></script>
```

Comment:
The `wpt-firefox-nightly-stability` check timed out once out of 10. Maybe a `long` timeout helps?

Reply:
Worth a shot. I'd love to find time for [a more intuitive solution](https://github.com/web-platform-tests/rfcs/pull/53)